src/components/SignUpPage/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ButtonGoogle from 'components/ButtonGoogle';
+import Card from 'components/Card'
+
+const SignUpPage: React.SFC<{}> = () => {
+  return (
+    <Card>
+      <p>Log in with one of the following:</p>
+      <a href={`https://accounts.google.com/o/oauth2/v2/auth?scope=https://www.googleapis.com/auth/userinfo.email&include_granted_scopes=true&state=state_parameter_passthrough_value&redirect_uri=http://localhost:3000/auth&response_type=token&client_id=${process.env.REACT_APP_GOOGLE_CLIENT_ID}`}><ButtonGoogle /></a>
```

Comment:
store this url as env i.e. REACT_APP_GOOGLE_AUTH_URL

Reply:
I noticed that this url is not working in incognito mode or when you are not logged to google

Reply:
when you are in incognito mode, you should be redirected to googla authorization page. Can you say something more?

Reply:
the same when you aren't logged

Reply:
looks like something was wrong with url, right now it's working flawlessly